chirp/drivers/tdh8.py

```diff
@@ -1081,8 +1081,11 @@ def get_memory(self, number):
 
         if _mem.get_raw(asbytes=False)[0] == "\xff":
             mem.empty = True
+            self.set_usedFlag(mem)
             return mem
 
+        self.set_usedFlag(mem)
+
```

Comment:
These change the image just because we fetched a memory out of it, which is not cool. That alters the state even though the UI says it's not modified and in general is not a good thing to do in something that should otherwise be a read-only operation. You wouldn't want a word processor modifying your document just because you opened it and didn't change anything...

Reply:
In this case, those bits being set is a form of corruption, as it is an invalid configuration. 

I suppose an argument could be made that this change is to catch and restore, but that's not our business.

Reply:
You can (should) be tolerant of corruption in `get_memory()`. You should not modify/fix memory until asked to update it I'd say. We have lots of cases where radios come from the factory with out-of-range values in fields where we need to be tolerant of them and take a default when and where we can. If there's no sane default, then raise an exception if we just need to signal it.